components/music/cards/Playlist.tsx

```diff
@@ -72,6 +74,9 @@ const Playlist = async ({
           {playlist.tracks.total === 1 ? "track" : "tracks"}
         </p>
       </div>
+      <div>
+        <PlayContext uri={playlist.uri} />
```

Comment:
Put this inside the `<div className="card-actions justify-end">` div, above the share icon instead.